consensus/service/src/config.rs

```diff
@@ -216,29 +216,29 @@ impl Config {
     }
 
     /// Get the configured minimum fee.
-    pub fn minimum_fees_map(&self) -> Result<Option<BTreeMap<TokenId, u64>>, String> {
+    pub fn fee_map(&self) -> Result<FeeMap, String> {
```

Comment:
Being able to always end with a `FeeMap` object that contains valid fee information and can easily be passed around is the main purpose of this PR.
This will allow me to pass it to `BlockchainApiService` so that it could hand the fee information to the clients.

Reply:
The goal is to guarantee that this information will be provided, as opposed to allowing it to be optional and forcing the client to figure it out for themselves?